Source_Code/TM_Backend/TeamMentor.Users/Users/Users_Creation.cs

```diff
@@ -239,6 +251,24 @@ public static Signup_Result createTmUserResponse(this TM_UserData userData, NewU
         }
 
         #region Sigup validations, shorter and specialized methods
+        private static Signup_Result ValidateEmailPattern(TMConfig config)
+        {
+            var sigupResponse = new Signup_Result
+            {
+                Signup_Status = Signup_Result.SignupStatus.Validation_Failed,
+                UserCreated = 0
+            };
+            var errorMessage = TMConfig.Current.TMErrorMessages.Email_Address_Is_Invalid;
+            if (config.showDetailedErrorMessages())
+            {
+                sigupResponse.Validation_Results.Add(new Validation_Results { Field = "Email", Message = errorMessage });
+            }
+            else
+                sigupResponse.Simple_Error_Message = config.TMErrorMessages.General_SignUp_Error_Message;
+
+            return sigupResponse;
+        }
+
         private static Signup_Result ValidatePasswordComplexity(TMConfig config)

```

Comment:
I was going to add a comment that this should also be an extension method, but just looked at https://github.com/michaelhidalgo/Dev/blob/3.6_Dev/Source_Code/TM_Backend/TeamMentor.Users/Users/Users_Creation.cs#L254-L328 and those functions really should be refactored into a couple simple methods (since the code is very similar)